src/services/menu-service.js

```diff
@@ -79,6 +88,156 @@ class MenuService {
       }
     }
   }
+
+  async createMarkdownContentFromNodes(nodes, level, lines = []) {
+    for (const node of nodes) {
+      if (node.title) {
+        lines.push(`###${'#'.repeat(level)} ${node.title}`);
+        lines.push('');
+      }
+
+      if (node.documentKeys.length) {
```

Comment:
Could documentKeys be undefined? safer would be node.documentKeys?.length

Reply:
They are always empty string if there are no keys, tested on the production dump